Formula/spidermonkey.rb

```diff
@@ -1,19 +1,13 @@
 class Spidermonkey < Formula
   desc "JavaScript-C Engine"
   homepage "https://developer.mozilla.org/en/SpiderMonkey"
-  url "https://archive.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz"
-  version "1.8.5"
-  sha256 "5d12f7e1f5b4a99436685d97b9b7b75f094d33580227aa998c406bbae6f2a687"
-  revision 1
+  url "https://people.mozilla.org/~sstangl/mozjs-38.2.1.rc0.tar.bz2"
+  version "38.2.1"
```

Comment:
Couldn't [find](https://github.com/Homebrew/legacy-homebrew/pulls?utf8=%E2%9C%93&q=is%3Apr+spidermonkey+author%3Adholm+is%3Aclosed+) anything older than April 2015 with a quick check.